core/src/smartcontracts/isi/world.rs

```diff
@@ -299,41 +298,14 @@ pub mod isi {
             state_transaction: &mut StateTransaction<'_, '_>,
         ) -> Result<(), Error> {
             let parameter = self.parameter;
-            let parameter_id = parameter.id.clone();
-
             let world = &mut state_transaction.world;
-            if !world.parameters.remove(&parameter) {
-                return Err(FindError::Parameter(parameter_id).into());
-            }
-
-            world.parameters.insert(parameter);
-
-            world.emit_events(Some(ConfigurationEvent::Changed(parameter_id)));
-
-            Ok(())
-        }
-    }
-
-    impl Execute for NewParameter {
-        #[metrics(+"new_parameter")]
-        fn execute(
-            self,
-            _authority: &AccountId,
-            state_transaction: &mut StateTransaction<'_, '_>,
-        ) -> Result<(), Error> {
-            let parameter = self.parameter;
-            let parameter_id = parameter.id.clone();
 
-            let world = &mut state_transaction.world;
-            if !world.parameters.insert(parameter) {
-                return Err(RepetitionError {
-                    instruction_type: InstructionType::NewParameter,
-                    id: IdBox::ParameterId(parameter_id),
-                }
-                .into());
+            if !world.executor_data_model.parameters.contains(&parameter.id) {
+                return Err(FindError::Parameter(parameter.id).into());
```

Comment:
what if the parameter being set is a core parameter?

Reply:
Replied in other thread.